package.json

```diff
@@ -92,6 +92,7 @@
     "cli-table": "^0.3.1",
     "commander": "^4.0.1",
     "configstore": "^5.0.1",
+    "cors": "^2.8.5",
```

Comment:
This is tricker than I thought, and I'm glad someone else solved it already. This adds [6KiB download size only](https://arve0.github.io/npm-download-size/#cors), mainly due to `vary` (but we'd need to handle that anyway). This is also already a dependency in Firebase Functions SDK.